netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
I know that, but I am not sure if we need this added feature right now :)

I would rather add version flag to each binary first and then figure/discuss usecases for having versions visible at one place for contiv-network service as a whole than just netmaster.